pkg/testutils/sensors/load.go

```diff
@@ -112,8 +112,33 @@ func mergeSensorMaps(maps1, maps2 []SensorMap, progs1, progs2 []SensorProg, t *t
 	return mapsReturn, progsReturn
 }
 
+func mergeInBaseSensorMaps(sensorMaps []SensorMap, sensorProgs []SensorProg, t *testing.T) ([]SensorMap, []SensorProg) {
```

Comment:
same with `mergeSensorMaps`: let's add the t *testing.T arg first.
Ideally, all test functions should be in _test.go files, but not sure if that's possible.

Reply:
yes, will move the arg

it's all under `pkg/testutils/sensors/load.go` so testing related